components/cli/vendor/github.com/docker/docker/client/client.go

```diff
@@ -260,8 +261,13 @@ func (cli *Client) NegotiateAPIVersionPing(p types.Ping) {
 		p.APIVersion = "1.24"
 	}
 
-	// if server version is lower than the current cli, downgrade
-	if versions.LessThan(p.APIVersion, cli.ClientVersion()) {
+	// if the client is not initialized with a version, start with the latest supported version
```

Comment:
This is not part of the upstream change. If it is needed then reference the upstream change from this PR.